ComfortZoneApp/app/src/main/java/com/example/comfortzone/MainActivity.java

```diff
@@ -8,6 +8,7 @@
 import androidx.fragment.app.FragmentManager;
 
 import android.content.Intent;
+import android.content.pm.PackageManager;
```

Comment:
@fionamei you can also create a module called activity and add all the activity classes in that module to keep the project structure clean.